scripts/environment-delete.sh

```diff
@@ -138,13 +142,17 @@ function emptyS3Bucket() {
     local bucket=$1
     local region=$2
     local delete_option=$3
-    printf "\n- Emptying bucket $bucket ... "
+
+    blank="                                                                                                                        "
```

Comment:
What is this var and why do we need it?

Reply:
To make display the operation progress in a smart way I am using the escape character `\r` that allow me to print a message on the same line as the previous message.
In order to update the progress displayed in one single line, I am printing as follow :  
`printf "\\r$blank\\r- Emptying bucket $bucket ... $i/$count" `

Precisely this instruction prints `$blanck` from the start of the current line of the console (it clears the line), then it displays the updated progress message.

Reply:
perfect, thanks for explaining! yeah in the picture I just noticed at the bottom of the screenshot the progression